### PR TITLE
Add unique package count to origins API endpoint

### PR DESCRIPTION
**Pull request by raskchanky**
tenor-153504077

Signed-off-by: Josh Black <raskchanky@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-protocol/protocols/originsrv.proto [hunk 1]
```diff
@@ -31,6 +31,7 @@ message Origin {
   optional string name = 2;
   optional uint64 owner_id = 3;
   optional string private_key_name = 4;
+  optional uint64 unique_package_count = 5;
```
Comment (chefsalim) [inline components/builder-protocol/protocols/originsrv.proto]: A couple of comments - this seems a bit out of place in Origin definition - I'm not sure this is the way we want to go. Can we look at adding to the stats API that has package count already instead?

Comment (raskchanky) [reply components/builder-protocol/protocols/originsrv.proto]: My first thought was the same as yours - does this belong in the Origin definition?  Seeing `private_key_name` in there made me think that we weren't being super strict about what went into the Origin definition.  The goal here was to give @elliott-davis the info he needed with a minimum of API calls.  I'll check out the stats API.

Comment (chefsalim) [reply components/builder-protocol/protocols/originsrv.proto]: Yeah I'm not sure private_key_name should be there either :(

**Comment by raskchanky** [comment]
@chefsalim will do

**Comment by raskchanky** [comment]
Given that making this change requires undoing literally everything I did in this PR, I'm just going to close this and open a different PR with the changes.

**Pull request closed without merging**